mne/viz/epochs.py

```diff
@@ -894,10 +894,13 @@ def plot_epochs(epochs, picks=None, scalings=None, n_epochs=20, n_channels=20,
                   scalebars_visible=False,
                   window_title=title,
                   xlabel='Epoch number')
-    fig = _browse_figure(**params)
+
+    fig = _get_browser(**params)
+
     fig._update_picks()
 
-    # make channel selection dialog, if requested (doesn't work well in init)
+    # make channel selection dialog,
+    # if requested (doesn't work well in init)
```

Comment:
Why the newline here?

Reply:
Good question, probably came with trying to adhere to <79 chars and the line looked long (but wasn't too long in fact).

Reply:
It's still two lines